### DISCUSSION TITLE
Guide for configuring Grbl - Portuguese

### DISCUSSION THREAD

**Discussion by Tribo-2dot0** [arkypita/LaserGRBL#1909] [General]
I saw the post in the FB group of the translation of the guide into German. I decided to translate it into Portuguese... it was posted there and I'm leaving it here too.
I don't know if there is a specific area, if so please move it.
[Grbl-Configuration-PT-BR.pdf](https://github.com/arkypita/LaserGRBL/files/9428346/Grbl-Configuration-PT-BR.pdf)
.

**Comment by arkypita** [comment]
Yeah, thank you!
I'll upload to LaserGrbl website too
